src/main/java/adris/altoclef/ConfigState.java

```diff
@@ -1,30 +1,36 @@
 package adris.altoclef;
 
+
 import baritone.altoclef.AltoClefSettings;
 import baritone.api.Settings;
-import baritone.process.MineProcess;
 import baritone.api.utils.RayTraceUtils;
+import baritone.process.MineProcess;
 import net.minecraft.client.MinecraftClient;
 import net.minecraft.entity.Entity;
 import net.minecraft.item.Item;
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.world.RaycastContext;
 
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Stack;
 import java.util.function.Predicate;
 
+
 /**
- * Represents a state of global config. It can be copied and reset
- * so that behaviour across tasks is consistent.
+ * Represents a state of global config. It can be copied and reset so that behaviour across tasks is consistent.
  */
 public class ConfigState {
 
-    private AltoClef _mod;
-
-    Stack<State> _states = new Stack<>();
+    private final AltoClef mod;
+    // TODO: 2021-05-22 remove stack
```

Comment:
Interesting, how come?

Reply:
the java `Stack` class is a bad implementation and bad practice to use.